src/AvaloniaEdit/AvaloniaEdit.csproj

```diff
@@ -33,7 +33,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Avalonia" Version="$(AvaloniaVersion)" />
+    <PackageReference Include="Avalonia" Version="0.10.0-preview5" />
```

Comment:
should be defined here:
https://github.com/AvaloniaUI/AvaloniaEdit/blob/master/Directory.Build.props#L5